### PR TITLE
feat: java oci example

### PR DESCRIPTION
**Pull request by alexeagle**
Note, still having a problem getting the app to run in the container

### CODE REVIEW

File: oci_java_image/test.yaml [hunk 1]
```diff
@@ -0,0 +1,5 @@
+schemaVersion: 2.0.0
+commandTests:
+  - name: 'test'
+    command: '/src/main/java/com/example/JavaLoggingClient'
```
Comment (alexeagle) [inline oci_java_image/test.yaml:4]: @thesayyn this is where we fail, it says the path doesn't exist even though I checked the image with `dive` and it's there and executable

Comment (alexeagle) [reply oci_java_image/test.yaml:4]: do we have to use command=java and entrypoint=something.jar? we want to use the bash wrapper stub I think

**Comment by alexeagle** [comment]
FYI @josephglanville

File: oci_java_image/BUILD.bazel [hunk 2]
```diff
@@ -0,0 +1,45 @@
+load("@rules_oci//oci:defs.bzl", "oci_image", "oci_tarball", "structure_test")
```
Comment (thesayyn) [inline oci_java_image/BUILD.bazel:1]: old version of rules_oci?

**Review by thesayyn — APPROVED** [review]

File: .github/workflows/ci.yaml [hunk 3]
```diff
@@ -61,10 +61,10 @@ jobs:
           - 'git_push'
           - 'go_workspaces'
           - 'jest'
-          - 'oci_go_image'
```
Comment (josephglanville) [inline .github/workflows/ci.yaml:64]: Is this change expected?

**Review by josephglanville — APPROVED** [review]
LGTM

File: oci_java_image/BUILD.bazel (hunk 2 above)
Comment (alexeagle) [reply oci_java_image/BUILD.bazel:1]: still the latest release :)
We should cut one before the actual RC and rollout here and other places to ensure we didn't miss something obvious

File: .github/workflows/ci.yaml (hunk 3 above)
Comment (alexeagle) [reply .github/workflows/ci.yaml:64]: yeah it was a duplicate line and was causing us to run that test twice

**Pull request merged**